locales/es-ES.json

```diff
@@ -71,20 +71,20 @@
 	"Twitter": "Twitter",
 	"About": "Sobre",
 	"Boostnote": "Boostnote",
-	"An open source note-taking app made for programmers just like you.": "Una aplicación para tomar notas de código abieto para programadores como tú.",
+	"An open source note-taking app made for programmers just like you.": "Una aplicación de código abierto para tomar notas hecho para programadores como tú.",
 	"Website": "Página web",
 	"Development": "Desarrollo",
 	" : Development configurations for Boostnote.": " : Configuraciones de desarrollo para Boostnote.",
 	"Copyright (C) 2017 - 2018 BoostIO": "Copyright (C) 2017 - 2018 BoostIO",
 	"License: GPL v3": "Licencia: GPL v3",
 	"Analytics": "Analítica",
-	"Boostnote collects anonymous data for the sole purpose of improving the application, and strictly does not collect any personal information such the contents of your notes.": "Boostnote recopila datos anónimos con el único propósito de mejorar la aplicación. No recopila ninguna información personal, como puede ser el contenido de sus notas.",
+	"Boostnote collects anonymous data for the sole purpose of improving the application, and strictly does not collect any personal information such the contents of your notes.": "Boostnote recopila datos anónimos con el único propósito de mejorar la aplicación, y de ninguna manera recopila información personal como el contenido de sus notas.",
 	"You can see how it works on ": "Puedes ver cómo funciona en ",
 	"You can choose to enable or disable this option.": "Puedes elegir activar o desactivar esta opción.",
-	"Enable analytics to help improve Boostnote": "Activa analítica para ayudar a mejorar Boostnote",
+	"Enable analytics to help improve Boostnote": "Activar analítica para ayudar a mejorar Boostnote",
```

Comment:
"analítica" could be replace with "datos estadísticos".

Reply:
Yes, "datos estadísticos" can fit better.

Reply:
And what about "Activar recopilación de datos"?

Reply:
Yes, that's even more understandable.